dali/imgcodec/decoders/libtiff/tiff_libtiff.cc

```diff
@@ -114,10 +114,12 @@ struct TiffInfo {
   uint16_t bit_depth;
   uint16_t orientation;
   uint16_t compression;
+  uint16_t photometric;
```

Comment:
Nitpick: the name sounds as if it's a boolean predicate - I'd expect a noun.

Reply:
Done